packages/office-ui-fabric-react/src/components/Rating/Rating.tsx

```diff
@@ -47,39 +51,55 @@ export class Rating extends BaseComponent<IRatingProps, IRatingState> {
+
+    if (this.props.aggregate) {
+      ratingContainer = <div className={ css('ms-Rating-container', styles.container) } role='img' alt={ this.props.rating.toString() } aria-labelledby={ `${this._labelId}-${this.props.rating}` }>
+        { this._getLabel(this.props.rating) }
+        { stars }
+      </div>;
+    } else {
+      ratingContainer = <div className={ css('ms-Rating-container', styles.container) } role='radiogroup' aria-labelledby={ this.props.ariaLabelId }>
+        { stars }
+      </div>;
+    }
+
     return <div className={ css('ms-Rating', this.props.className, {
       ['ms-Rating--large ' + styles.rootIsLarge]: this.props.size === RatingSize.Large
     }) } role='application'>
-      <div className={ css('ms-Rating-container', styles.container) } role='radiogroup' aria-labelledby={ this.props.ariaLabelId }>
-        { stars }
-      </div>
+      { ratingContainer }
     </div>;
   }
 
   private _renderStar(rating: number): JSX.Element {
     const inputId = `${this._id}-${rating}`;
+    const flooredRating = Math.floor(this.state.rating);
+    const starSize = this.props.aggregate && rating > this.state.rating && rating - 1 < this.state.rating ? this.state.rating - flooredRating : 1;
 
     return <div className={ css('ms-Rating-star', styles.star, {
       ['is-selected ' + styles.starIsSelected]: rating <= this.state.rating,
       ['is-inFocus ' + styles.starIsInFocus]: rating === this.state.focusedRating,
-      ['is-disabled ' + styles.starIsDisabled]: this.props.disabled
+      ['is-disabled ' + styles.starIsDisabled]: !this.props.aggregate && this.props.disabled,
+      ['is-aggregate ' + styles.starIsAggregate]: this.props.aggregate
     }) } key={ rating }>
-      <input
-        className={ css('ms-Rating-input', styles.input) }
-        type='radio'
-        name={ this._id }
-        id={ inputId }
-        value={ rating }
-        aria-labelledby={ `${this._labelId}-${rating}` }
-        disabled={ this.props.disabled }
-        checked={ rating === this.state.rating }
-        onChange={ this._onChange.bind(this, rating) }
-        onFocus={ this._onFocus.bind(this, rating) }
-        onBlur={ this._onBlur.bind(this, rating) }
-      />
+      { !this.props.aggregate &&
+        <input
+          className={ css('ms-Rating-input', styles.input) }
+          type='radio'
+          name={ this._id }
+          id={ inputId }
+          value={ rating }
+          aria-labelledby={ `${this._labelId}-${rating}` }
+          disabled={ this.props.disabled }
+          checked={ rating === this.state.rating }
+          onChange={ this._onChange.bind(this, rating) }
+          onFocus={ this._onFocus.bind(this, rating) }
+          onBlur={ this._onBlur.bind(this, rating) }
+        />
+      }
       <label className={ css('ms-Rating-label', styles.label) } htmlFor={ inputId }>
-        { this._getLabel(rating) }
-        { this._getIcon() }
+        { !this.props.aggregate && this._getLabel(rating) }
```

Comment:
htmlFor will always have this value: inputId = `${this._id}-${rating}`;
We are just skipping render of label in case of aggregate control so that we just have one single value for screen readers